src/reconcile/rec_row.c

```diff
@@ -78,6 +78,8 @@ __rec_cell_build_int_key(
     /* Create an overflow object if the data won't fit. */
     if (size > btree->maxintlkey) {
         WT_STAT_DATA_INCR(session, rec_overflow_key_internal);
+        if (WT_IS_HS(btree))
```

Comment:
Already there exists a data store stat that indicates how many overflow keys are present. As these stats are specific to each data store, we are already tracking it for history store also. Do we really need a new stat here?